.circleci/config.yml

```diff
@@ -103,6 +103,26 @@ jobs:
           command: |
             echo "Deploying to Heroku. To see progress, go to: https://dashboard.heroku.com/apps/$HEROKU_APP_NAME_PRODUCTION/activity"
             curl -sf -X POST -m 900 https://heroku-deploy.torchbox.com/$HEROKU_APP_NAME_PRODUCTION/$CIRCLE_SHA1?key=$DEPLOYMENT_KEY -d "Content-Length: 0"
+
+  copy_prod_to_staging:
+    docker:
+      - image: buildpack-deps:trusty
+    steps:
+      - checkout
+      - run:
+          name: update packages
+          command: 'apt-get update -y'
+      - run:
+          name: install curl, jq
+          command: 'apt-get install -y curl jq'
+      - run:
+          name: Send POST request to FlightPath
+          command: |
+            export JOB_STATUS=$(curl -f -X POST -m 900 -H "Authorization: Token $FLIGHTPATH_AUTH_KEY" -d "source_key=$DEPLOYMENT_KEY&destination_key=$DEPLOYMENT_KEY&copy_media=1&from_backup=1" "https://flightpath.torchbox.com/copy/$HEROKU_APP_NAME_PRODUCTION/$HEROKU_APP_NAME_STAGING/")
```

Comment:
Im pretty sure this needs to target HEROKU_APP_NAME_DEV right? Beause we don't use the staging site still https://intranet.torchbox.com/our-clients/royal-college-of-art/#working-with-dev-and-staging

Reply:
This might be why data isn't sanitized on dev?

Reply:
Resolved in https://github.com/torchbox/rca-wagtail-2019/pull/871/commits/7196f4a975eeb9fa3bd986b880178311d90d29b1